README.md

```diff
@@ -105,30 +105,27 @@ serve --cors
 The data will be available at `http://localhost:3000` over CORS, with no path prefix. For example, device data will be at `http://localhost:3000/devices.json`.
 
 
-### Deploying to Staging (18F specific)
-Pushes to the 18f-pages branch are reflected in the staging environment https://pages.18f.gov/analytics.usa.gov/. All changes can be seen here prior to production release.
+### Deploying the app
 
-### Deploying the app to production
+To deploy to **analytics.usa.gov** after building the site with the details in `_config.yml`:
 
-In production, the site's base URL is set to `https://analytics.usa.gov` and the data's base URL is set to `https://analytics.usa.gov/data/live`.
-
-To deploy this app to `analytics.usa.gov`, you will need authorized access to 18F's Amazon S3 bucket for the project.
+```bash
+make deploy_production
+```
 
-To deploy the site using `s3cmd`, production settings, and a **5 minute cache time**, run:
+To deploy to **analytics-staging.app.cloud.gov** after building the site with the details in `_config.yml` and `_staging.yml`:
 
 ```bash
-make deploy
+make deploy_staging
 ```
 
-**Use the full command above.** The full command ensures that the build completes successfully, with production settings, _before_ triggering an upload to the production bucket.
-
 
 ### Environments
 
 | Environment | Branch | URL |
 |-------------| ------ | --- |
-| Staging | 18f-pages  | https://pages.18f.gov/analytics.usa.gov/  |
-| Production | 18f-pages | https://www.analytics.usa.gov  |
+| Production | master | https://analytics.usa.gov |
+| Staging | master | https://analytics-staging.app.cloud.gov |
```

Comment:
I haven't created a staging-specific branch. Leaving the desired branch/deploy management process up to future devs.